karma.conf.js

```diff
@@ -77,7 +77,7 @@ module.exports = function(config) {
         ]
       },
       node: {
-        Buffer: false,
+        Buffer: true,
```

Comment:
Since using forge in tests, revert this back to false.

Reply:
Based on other comment, and this is just in karma, all's good.  But might as well remove the line since true is the default.

Reply:
I would prefer to be explicit about it since I'm no karma expert.  It was fortunate that this entry was here to let me know that I needed to change it.